### DISCUSSION TITLE
use only part of the filename when concatenating images

### DISCUSSION THREAD

**Discussion by probiner** [ImageMagick/ImageMagick#4538] [Help]
A command like this:
`montage -mode concatenate img.left.png img.front.png img.top.png img.max.png -gravity southwest -fill white -pointsize 32 -annotate 0 '%t' result.png`
allows me to me get multiple image files stitched into a single image with unique label for each. But what I would like to have instead of "img.left" and "img.front", etc would be "left", "front" and so on, in the label. Is there anyway to tell imagemagik to only care for a part of a image filename for the annotation?

I would want to avoid overwriting the individua inputl files, so adding the labels to the individual files before _montage_ is not ideal.

Thanks
prb

![image](https://user-images.githubusercontent.com/1182548/144528220-f42465e4-a0a2-4e7a-b053-a8c14f5bff4a.png)

**Comment by urban-warrior** [comment]
Use `-label`, for example:

```
montage -label Rose rose: -label Wizard wizard: image.png
````
